pom.xml

```diff
@@ -163,8 +163,8 @@
     <os-maven-plugin.version>1.5.0.Final</os-maven-plugin.version>
     <groovy-maven-plugin.version>2.0</groovy-maven-plugin.version>
     <license-maven-plugin.version>1.14</license-maven-plugin.version>
-    <maven-surefire-plugin.version>2.20.1</maven-surefire-plugin.version>
-    <maven-failsafe-plugin.version>2.20.1</maven-failsafe-plugin.version>
+    <maven-surefire-plugin.version>2.21.0</maven-surefire-plugin.version>
```

Comment:
Did you mean to do this?

If so, can you update build.gradle to match this version:
https://github.com/apache/beam/blob/d8cb0e6098370dc5059834ea82c5f6fa22f6a91b/build.gradle#L159

Reply:
Yes, it was intended. I was not aware gradle was using that plugin too. Thanks for the info. I will fix that one too.